patches/minecraft/net/minecraft/client/MouseHelper.java.patch

```diff
@@ -40,3 +40,14 @@
              }, "mouseDragged event handler", iguieventlistener.getClass().getCanonicalName());
           }
  
+@@ -224,6 +232,10 @@
+       return this.field_198039_d;
+    }
+ 
++   public boolean isMiddleDown() {
```

Comment:
Please put new methods at the bottom of the file, after a comment like this:
` /* ======================================== FORGE START =====================================*/`